chia/full_node/full_node_api.py

```diff
@@ -1498,14 +1504,20 @@ async def register_interest_in_coin(
 
         if peer.peer_node_id not in self.full_node.peer_sub_counter:
             self.full_node.peer_sub_counter[peer.peer_node_id] = 0
-        max_items = self.full_node.config.get("max_subscribe_items", 200000)
+        if self.is_trusted(peer):
+            max_items = self.full_node.config.get("trusted_max_subscribe_items", 2000000)
+        else:
+            max_items = self.full_node.config.get("max_subscribe_items", 200000)
         for coin_id in request.coin_ids:
+
+            # check subscription limit
+            if self.full_node.peer_sub_counter[peer.peer_node_id] >= max_items:
+                self.log.warning(f"wallet {peer.peer_node_id} exceeded max coin subscription limit")
```

Comment:
I think this should be an `INFO`. The person running the full node probably doesn't care *that* much about one of its wallet peers

Reply:
same as above